Automattic/wp-calypso
# Happy Chat: Set up and send some Custom Fields

#### Changes proposed in this Pull Request

Implements light infrastructure to send Custom Fields into Happy Chat, and sends a couple of the Contact Form options through. More on Custom Fields here: pbvpgB-UE-p2

#### Testing instructions

- Sign into Happy Chat and go green (in staging or local env, make sure your local Calypso is going to the same place)
- Go to http://calypso.localhost:3000/help/contact and start a chat
- In the Happy Chat HUD you should see `howCanWeHelp` and `howYouFeel` custom fields corresponding to the options you chose
- Fill the form a couple times, changing the form options — ensure the HUD reflects the options you chose
